docs/docs/metrics/model-quality-metrics.md

```diff
@@ -85,4 +85,16 @@ It can be employed to define a heuristic that measures a model’s uncertainty a
 Like before, the lower the image's score, the more “confused” the model is.
 As a result, data samples with lower variance score should be offered for annotation.
 
-Implementation on [GitHub](https://github.com/encord-team/encord-active/blob/main/src/encord_active/lib/metrics/acquisition_functions.py)
+Implementation on [GitHub](https://github.com/encord-team/encord-active/blob/main/src/encord_active/lib/metrics/acquisition_metrics/acquisition_functions.py)
+
+
+### Average Frame Score {#average-frame-score}
```

Comment:
"Average Frame Score" is very general. It could basically be anything. Let's be more specific. For example, "Mean detection confidence" or something like that.

Reply:
'mean object score' maybe?